src/sample_package/src/MyNode.cpp

```diff
@@ -15,18 +15,19 @@ MyClass::MyClass(int argc, char **argv, std::string node_name) {
     ros::NodeHandle private_nh("~");
 
     // Obtains character from the parameter server (or launch file), sets '!' as default
-    std::string parameter_name = "my_node/character";
+    std::string parameter_name = "character";
     std::string default_character = "!";
-    SB_getParam(nh, parameter_name, suffix, default_character);
+    SB_getParam(private_nh, parameter_name, suffix, default_character);
+
 
     // Setup Subscriber(s)
     std::string topic_to_subscribe_to = "subscribe_topic";
-    int refresh_rate = 10;
-    my_subscriber = nh.subscribe(topic_to_subscribe_to, refresh_rate, &MyClass::subscriberCallBack, this);
+    int queue_size = 10;
+    my_subscriber = nh.subscribe(topic_to_subscribe_to, queue_size, &MyClass::subscriberCallBack, this);
 
     // Setup Publisher(s)
     std::string topic = private_nh.resolveName("publish_topic");
-    uint32_t queue_size = 1;
+    queue_size = 10;
```

Comment:
isn't this a redundant line? but otherwise looks good!

Reply:
It is redundant, but I thought it best to set it again here so people don't get the idea that the queue sizes need to be the same. I'll change the value so it's a bit more clear